spec/conjurctl/configuration_show_spec.rb

```diff
@@ -37,4 +37,16 @@
       "error: Unknown configuration output format 'invalid'\n"
     )
   end
+
+  it "outputs an error when configuration is invalid" do
+    _stdout, stderr, status = Open3.capture3(
+      "CONJUR_TRUSTED_PROXIES=boop conjurctl configuration show"
+    )
```

Comment:
Not a big deal, but this feels more like an end to end test, rather than a unit test.  I'd expect it to exist, but not in this spec file...

Reply:
Yep, all of these tests need to be moved but it's a problem for another day.